### DISCUSSION TITLE
add a CFGDB_SCHEMA setting to the make process

### DISCUSSION THREAD

**Discussion by pljakobs** [mikee47/ConfigDB#30] [Ideas]
currently, the build process just processes all .cfgdb files in the project directory.
To be more in line with how HWCONFIG is handled, I suggest that we add a setting to select a specific .cfgdb file. 
afaiu, using "include" within the .configdb file, we can still create multi-file configurations.

**Comment by mikee47** [comment]
Not quite the same as there can only be *one* hardware config for an application, whilst there can be multiple databases. If the application has multiple configurations, why would they only want to build one of them? What are the use-cases?

**Reply by pljakobs** [reply]
ok. my "Denkfehler" as we say - "fallacy on thinking" seems to be the English term. disregard, please, you're right.

**Reply by mikee47** [reply]
Another wonderful word! I shall have to work that one into conversation - thank you!
